.gitlab-ci.yml

```
@@ -173,6 +173,13 @@ test:backend:static:
 test:backend:validate-open-api:
   stage: test
   needs: []
+  rules:
+    - changes:
+        paths: ["backend/**/*.yml"]
```

This rule would also target compose files and certain test yaml files. Could we make the rule slightly more specific?
```suggestion
        paths: ["backend/**/docs/*.yml", "backend/**/api/*.yaml"]
```